packages/common-library/src/common_library/unset.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any
+
+
+class UnSet:
+    INSTANCE: "UnSet"
+
+
+UnSet.INSTANCE = UnSet()
```

Comment:
I like more a simple `unset`  object.  At the end of the day the helper below looks for a instance of UnSet. I do not wee why we have to create a unique instance for it. :-)